content/blog/2023/2022blog.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "2022 Progress!"
+author: "ghostway"
+published: 2023-01-14
+---
+
+2022 has been a great year for Leela. A lot of new contributors appeared and made significant improvements; overall, Leela has become considerably stronger and more interesting.
+<!--more-->
+
+While 2021 had [ceres](https://lczero.org/blog/2021/01/announcing-ceres/) written and small (but exciting) improvements to the training pipeline,
+in 2022 there were huge changes to how the search was conducted, network architecture, available backends and more.
```

Comment:
Style comment:
* "2022 brought us huge changes to the search, ..."
(trying to get rid of excessive passive case)

Reply:
I think I resolved that, even if indirectly